CHANGELOG.md

```diff
@@ -0,0 +1,256 @@
+# Change Log for AWS IoT Device SDK for Embedded C
+
+## 202009.00 (September 2020)
+
+This release is **NOT** backwards compatible with the v3 version of AWS IoT Device SDK for Embedded C.
```

Comment:
Does Amit want this at the end of the section?

Reply:
Yes pls, same as what we did in release candidate branch.

Reply:
Please make sure this file is same as the file in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1206.

Reply:
Not https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1206, but pls make sure this file is exactly same as its version from release-candidate.